_posts/2016-05-24-SpringIO16-Spring-Rest-Docs.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Spring I/O 16: Test-driven documentation with Spring REST Docs'
+image: /img/springio.jpg
+tags: [Spring IO,Spring,Conference]
+category: conference
+comments: true
+---
+
+## Spring IO 2016
+The main focus this year was definitely about Cloud, Reactive and MicroServices.
+But to complete this story we will need to document the REST API. 
```

Comment:
We need to document **the** REST API? We were talking about this year's focus and now I have to document **the** REST API :)?

I'd rephrase this to, "but it is important not to forget other topics, like documentation, ..." or something similar.